docs/app/documentation/component-docs/alert/examples/alert-component-as-content-example.component.ts

```diff
@@ -14,6 +14,7 @@ export class AlertComponentAsContentExampleComponent {
     openFromComponent() {
         this.alertService.open(AlertContentComponent, {
             type: 'warning',
+            minWidth: '400px',
```

Comment:
Maybe just put this one down to 300, even if it's the default. 400px will break on some phone screens which defeats the purpose of this feature. The rest seems pretty good to me though. Eventually we'll want to address #865 but that can wait for now.

Reply:
Hi @MattL75 I changed 400px to 300px in example.